packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -7,6 +7,8 @@ category: management
 
 <ApiLifecycle access="ea" />
 
+> **Note:** This API is currently being enhanced, accessible at ```/api/v1/idp/myaccount```. We encourage you to use this endpoint if you plan to use this API. Endpoints utilizing the existing endpoint (```/api/v1/myaccount```) will be deprecated. We encourage you to use ```/api/v1/idp/myaccount``` if you plan to use this API.
```

Comment:
@justinabrokwah-okta When do we plan on publishing this note? If it is going out at the same time as the new docs, we shouldn't use the future tense (see my suggested edits).
```suggestion
> **Note:** The MyAccount API is being enhanced, accessible at `/api/v1/idp/myaccount`. Use this endpoint if you plan to use the MyAccount API. The existing endpoint (`/api/v1/myaccount`) is deprecated.
```

Reply:
@brentschaus-okta So we're working on a change which enables customers that use the existing MyAccount API to continue doing so; when that change goes out, we want this note to go out at the same time.

I don't have an ETA but I agree on changing the tense, I'll make that change